.github/workflows/dev-build-on-linter-commit.yml

```
@@ -0,0 +1,73 @@
+name: Scheduled check and build on new zlint commits
+
+on:
+  schedule:
+    # Run Hourly
+    - cron: '0 * * * *'
+  workflow_dispatch:
+  
+jobs:
+  check_release:
+    runs-on: ubuntu-latest
+    name: Daily check for new zlint release
+
+    steps:
+    - name: Checkout this repo
+      uses: actions/checkout@v4
+
+    - name: Setup git
+      run: |
+          git config --global user.email "gh@pkimet.al"
+          git config --global user.name "PKIMetal"
```

```suggestion
          git config --global user.name "pkimetal"
```

The project's name is "pkimetal".  No capitals.  No whitespace.  :-)